views/dashboard.ejs

```diff
@@ -5,64 +5,39 @@
-		<p id="dashb-balance">Your points balance is: <%= rider[0].pointBalance %></p>
+		<% riders.forEach(function(riders) { %>
+  		<p id="dashb-name">Hi <%= riders.name %>!</p>
+			<p id="dashb-balance">Your points balance is: <%= riders.pointBalance %></p>
+		<% }) %>
 	</div>
 
-<!-- Start Rewards -->
 	<div class="row">
-<!-- One Day Free Park -->
-	  <div class="col-sm-6 col-md-6">
-	    <div class="thumbnail text-center">
-		<br>
-	      <i class="fa fa-bus fa-5x" aria-hidden="true"></i>
-	      <div class="caption">
-	        <h3 class="text-center">One Day Free<br>Metro Ride</h3>
-	        <p class="text-center">5 Points</p>
-	        <p class="text-center"><a href="#" class="btn btn-default" id="oneday-redeem" role="button">Redeem</a></p>
-	      </div>
-	    </div>
-	  </div>
-<!-- One Week Free Park -->
-		<div class="col-sm-6 col-md-6">
-		  <div class="thumbnail text-center">
-		  <br>
-			<i class="fa fa-bus fa-5x" aria-hidden="true"></i>
-			<div class="caption">
-			  <h3 class="text-center">One Week Free<br>Metro Ride</h3>
-			  <p class="text-center">30 Points</p>
-			  <p class="text-center"><a href="#" class="btn btn-default" id="week-redeem" role="button">Redeem</a></p>
-			</div>
-		  </div>
-		</div>
-<!-- Uber Lyft -->
-		<div class="col-sm-6 col-md-6">
-		  <div class="thumbnail text-center">
-		  <br>
-			<i class="fa fa-car fa-5x" aria-hidden="true"></i>
-			<div class="caption">
-			  <h3 class="text-center">One Free Uber<br>or Lyft Ride</h3>
-			  <p class="text-center">(Under 5 miles)<br>50 Points</p>
-			  <p class="text-center"><a href="#" class="btn btn-default" id="uber-redeem" role="button">Redeem</a></p>
-			</div>
-		  </div>
-		</div>
-<!-- Car / Bike Park -->
-		<div class="col-sm-6 col-md-6">
-		  <div class="thumbnail text-center">
-		  <br>
-			<i class="fa fa-product-hunt fa-5x" aria-hidden="true"></i>
-			<div class="caption">
-			  <h3 class="text-center">1 Day Free Car Parking<br>or Bike Storage</h3>
-			  <p class="text-center"><br>50 Points</p>
-			  <p class="text-center"><a href="#" class="btn btn-default" id="parking-redeem" role="button">Redeem</a></p>
-			</div>
-		  </div>
-		</div>
-<!-- Img -->
+		<h2>Redeem Your Points - Link Version</h2>
+	  <p>Displays all Rewards; links to redeem page to see each point displayed individually</p>
+	  <ul>
+	  <% rewards.forEach(function(reward) { %>
+	    <li><a href="/dashboard/<%= reward.id %>"><%= reward.rewardType %></a>:</li>
+	  <% }) %>
+	  </ul>
 	</div>
-<!-- End Rewards -->
 
-</div>
+	<h2>Redeem Your Points - Form Version</h2>
+	<p>Displays all Rewards in a dropdown menu; SHOULD redirect to redeem page when "submit" button is clicked. But when this happens, no data is displayed</p>
+	<form action="/redeem" method="post">
+	  <div class="form-group">
+			<input type="hidden" name="" value="">
+	  </div>
 
+  <div class="form-group">
+    <label for="reward">Reward</label>
+    <select name="rewardType" id="points">
```

Comment:
Are you getting rid of the existing design of the dashboard? If you want to keep the existing design, just replace the "redeem" button with a form that has a hidden field. The hidden field will contain the data you want to pass to the `post` route.